src/org/skypro/skyshop/product/BestResultNotFound.java

```diff
@@ -0,0 +1,8 @@
+package org.skypro.skyshop.product;
+
+public class BestResultNotFound extends RuntimeException {
```

Comment:
исключения лучше называть с Exception на конце
```suggestion
public class BestResultNotFoundException extends RuntimeException {
```